homeassistant/components/logger.py

```diff
@@ -15,6 +15,7 @@
 
 DATA_LOGGER = 'logger'
 
+SERVICE_SET_DEFAULT_LEVEL = 'set_default_level'
```

Comment:
I would actually rename this to `set_level`. Since it is not the default... we changing it right now. The default is in the config... right?

Reply:
The default is *optional* when setting up `logger:` in the config. What would be the reason behind using `set_level` instead of `set_default_level` (`set_level` already exists, but does not allow you to change the log level for namespaces that do not have a level set).